lib/action_tracker/matchers/track_matcher.rb

```diff
@@ -0,0 +1,18 @@
+RSpec::Matchers.define :track do |expected|
+  match do |actual|
+    actual.send(:tracker_instance).respond_to?(expected)
+  end
+
+  description do
+    "track #{expected}."
+  end
+
+  failure_message do |actual|
+    "expected that #{actual} track #{expected} action, but it didn't."
```

Comment:
O que acha de mudar a mensagem pra _"expected #{actual} to track #{expected} action, but it didn't."_